index.html

```diff
@@ -178,27 +199,33 @@ <h3 class="project_heading">Professional Art Printing Data</h3>
       </section>
     </div>
 
-    <section class="about_myself">
+    <section class="about_myself" id="about">
       <div class="about_me">
-        <h1 class="about_myself_heading">About me</h1>
-        <p class="about_me_description">
-          Hello I'm a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-
-        <button class="get_my_resume button">Get My Resume</button>
+        <div class="aboutme_description">
+          <h1 class="about_myself_heading">About me</h1>
```

Comment:
- [ ] This "about me" headline should be a little larger (width-wise).   Kindly let's fix it so the left element takes some more of the page.

![image](https://user-images.githubusercontent.com/57421823/149313096-b1a84e41-d6b1-4e17-82bb-d725a0d212ab.png)